omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/ConversionUtil1_9Test.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web;
+
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.ConceptName;
+import org.openmrs.Drug;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.representation.CustomRepresentation;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.EncounterResource1_8;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+import java.util.List;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class ConversionUtil1_9Test extends BaseModuleWebContextSensitiveTest {
+
+
+    @Test
+    public void convertToRepresentation_shouldNotFailIfUnableToParseObsValueUsingCustomRep() {
```

Comment:
I would say what this really asserts, something like: `convertToRepresentation_shouldConvertObsDrugValueAsNull`.